formidable/forms/field_builder.py

```diff
@@ -97,13 +97,15 @@ def get_disabled(self):
     def get_initial(self):
         if self.field_is_dict and 'defaults' in self.field:
             defaults = self.field['defaults']
+            defaults = list(filter(None, defaults))
```

Comment:
same remarks for the "elif" below... removing this filter won't change the test results.

Reply:
and the code might be simplified with:

```python
defaults = [x for x in self.field["defaults"] if x]
```

Reply:
I have been unable to reproduce this use case, I removed this line.